app/src/main/java/org/sopt/korailtalk/core/designsystem/component/dialog/KorailTalkBasicDialog.kt

```diff
@@ -0,0 +1,164 @@
+package org.sopt.korailtalk.core.designsystem.component.dialog
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import org.sopt.korailtalk.core.common.util.extension.noRippleClickable
+import org.sopt.korailtalk.core.designsystem.component.button.DialogCancelButton
+import org.sopt.korailtalk.core.designsystem.component.button.DialogConfirmButton
+import org.sopt.korailtalk.core.designsystem.component.button.DialogSingleButton
+import org.sopt.korailtalk.core.designsystem.theme.KorailTalkTheme
+
+@Composable
+fun KorailTalkBasicDialog(
+    onDismissRequest: () -> Unit,
+    modifier: Modifier = Modifier,
+    message: String,
+    buttons: @Composable BoxScope.() -> Unit,
+) {
+    Dialog(onDismissRequest = onDismissRequest) {
+        KorailTalkBasicDialogContent(
+            message = message,
+            buttons = buttons,
+            modifier = Modifier
```

Comment:
상위에서 받아온 modifier 전달하게끔 해주시면 좋을 거 같아용
```suggestion
            modifier = modifier
```